git/src/main/kotlin/org/sonarlint/intellij/git/GitRepo.kt

```diff
@@ -74,7 +74,13 @@ class GitRepo(private val repo: GitRepository, private val project: Project) : V
     }
 
     private fun distance(project: Project, repository: GitRepository, from: String, to: String): Int? {
-        val mergeBase = GitHistoryUtils.getMergeBase(project, repository.root, from, to) ?: return null
```

Comment:
Can't we add a ProgressIndicator or Job?

Reply:
I'm not totally sure if it's an issue on our side, or a platform issue, given this [thread](https://youtrack.jetbrains.com/issue/IJPL-148354/WSL-GitLogUtil-There-is-no-ProgressIndicator-or-Job-in-this-thread-the-current-job-is-not-cancellable.#focus=Comments-27-9754277.0-0). If that's a platform issue, it should not be fixed by us. If that's an issue from us, it should not be fixed by a ProgressIndicator or a Job, it's an issue with the threading scope (coroutines) I believe

Reply:
Given that this error is simply a "loud error", I'd tend on keeping things simple and just catching the error